.travis.yml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+language: node_js
+os:
+  - windows
+  - linux
+
+node_js:
+  - "10"
```

Comment:
Would be good to have an entry for the exact version we use in Runtime, `10.19` for our custom blackbox images and apparently `10.15` for the standard nodejs kind.

Could replace 10 or be an additional entry.

Also add the latest, 13?

Reply:
(The new latest is 14 - Also, I would only use LTS (even) versions)

Reply:
I will add that. Btw, the reason I used "10" and "12" is because I used an example travis config from aio: https://github.com/adobe/aio-lib-template/blob/master/.travis.yml